docs/book/src/TODO.md

```diff
@@ -1,8 +0,0 @@
-# TODO
```

Comment:
What is your motivation to remove this one?

Reply:
this file is a plceholder that is not used yet, just that :)

Reply:
It is used when we face 404, see: https://github.com/kubernetes-sigs/kubebuilder/blob/ce24b89a23fb8a51be78b115c15fd0f08975b46a/docs/book/src/404.md?plain=1

Reply:
As other places see; https://github.com/search?q=repo%3Akubernetes-sigs%2Fkubebuilder%20TODO.md&type=code